android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -115,6 +127,29 @@ public Snapshot(View view) {
     }
     originXByParent = view.getLeft();
     originYByParent = view.getTop();
+    borderRadius = getBorderRadius(view);
+  }
+
+  private float getBorderRadius(View view) {
+    if (view.getBackground() != null) {
+      Drawable background = view.getBackground();
+      if (background instanceof ReactViewBackgroundDrawable) {
```

Comment:
If this check fails, we'll effectively return 0. Do you think it's possible that `background instanceof ReactViewBackgroundDrawable` is false but `view instanceof ReactImageView` is true? In such case, we could change `else if` to `if` below.